scripts/mobile-smoke-test.sh

```
@@ -0,0 +1,168 @@
+log_warn() { echo -e "${YELLOW}[⚠]${NC} $1"; }
+log_error() { echo -e "${RED}[✗]${NC} $1"; }
+
+check_prerequisites() {
+  log_info "Checking prerequisites..."
+  
+  if [ ! -f "$CORVUS_BINARY" ]; then
+    log_error "Corvus binary not found at $CORVUS_BINARY"
+    log_info "Run: make rust-build"
+    exit 1
+  fi
+  log_success "Corvus binary exists ($(ls -lh $CORVUS_BINARY | awk '{print $5}'))"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)